### PR TITLE
Add ClamAV signatures check

### LINKED ISSUE #118: Problem: clamav-daemon needs to wait for the signature files after initial installation
Version of the documentation: 1.7

Page (and section, if applicable) where the issue occurs: 

Install Ubuntu: 12. Start the remaining services

Description of the issue: 

See https://github.com/artefactual/archivematica/issues/999

The clamav-daemon service needs the signature file created by clamav-freshclam to start. Sometimes it takes longer than 1 minute the first time clamav-freshclam is started. 

This is a Clamav issue, but this behaviour can be documented on archivematica-docs.

### PR DESCRIPTION
**Pull request by mamedin**
Fixes #118

### CODE REVIEW

File: admin-manual/installation-setup/installation/install-centos.rst [hunk 1]
```diff
@@ -270,9 +270,55 @@ Installation instructions
...
+     repository, running:
+
+     .. code:: bash
+
+        sudo yum install -y wget
+        sudo -u root bash -c " \
+        wget -nc -nv -P /var/lib/clamav https://github.com/artefactual-labs/clamav-files/raw/master/main.cvd
+        wget -nc -nv -P /var/lib/clamav https://github.com/artefactual-labs/clamav-files/raw/master/daily.cvd
+        wget -nc -nv -P /var/lib/clamav https://github.com/artefactual-labs/clamav-files/raw/master/bytecode.cvd
+        sudo chown clamupdate:clamupdate /var/lib/clamav/*cvd
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-centos.rst]: sudo sudo? I guess we don't need that?

File: admin-manual/installation-setup/installation/install-centos.rst [hunk 2]
```diff
@@ -270,9 +270,55 @@ Installation instructions
...
+        ";
+
+   * Enable and Start ClamAV daemon:
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-centos.rst]: s/Start/start

File: admin-manual/installation-setup/installation/install-centos.rst [hunk 3]
```diff
@@ -270,9 +270,55 @@ Installation instructions
...
+
+     .. code:: bash
+
+        sudo systemctl is-active clamav-daemon
+
+     When the service is up, the output is:
+
+     .. code:: bash
+
+        active
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-centos.rst]: You could replace this with an inline literal.

> When the service is up, the output is: `active`.

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 4]
```diff
@@ -157,12 +157,53 @@ Ubuntu 16.04 (Xenial) installation instructions
        sudo service elasticsearch restart
        sudo update-rc.d elasticsearch defaults 95 10
 
-12. Start the remaining services
+12. Check ClamAV signatures (optional). To run the clamav-daemon service, it
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-ubuntu.rst]: s/Check/Check out? Or maybe Download?

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 5]
```diff
@@ -157,12 +157,53 @@ Ubuntu 16.04 (Xenial) installation instructions
...
+    needs the signature files created by clamav-freshclam to start. The first
+    time clamav-freshclam is started it can sometimes take longer than 1
+    minute.  Please, make sure that the following files have been downloaded at
+    ``/var/lib/clamav`` directory before starting the ClamAV daemon:
+
+    - bytecode.cvd
+    - daily.cld or daily.cvd
+    - main.cvd
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-ubuntu.rst]: ^ they could be inline literals 

```
    - :file:`bytecode.cvd`
    - :file:`daily.cld` or :file:`daily.cvd`
    - :file:`main.cvd`
```

Even better using `:file:` (http://www.sphinx-doc.org/en/stable/markup/inline.html).

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 6]
```diff
@@ -157,12 +157,53 @@ Ubuntu 16.04 (Xenial) installation instructions
        sudo service elasticsearch restart
        sudo update-rc.d elasticsearch defaults 95 10
 
-12. Start the remaining services
+12. Check ClamAV signatures (optional). To run the clamav-daemon service, it
+    needs the signature files created by clamav-freshclam to start. The first
+    time clamav-freshclam is started it can sometimes take longer than 1
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-ubuntu.rst]: L161 and L162: inline literal for clamav-freshclam?

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 7]
```diff
@@ -157,12 +157,53 @@ Ubuntu 16.04 (Xenial) installation instructions
...
+    minute.  Please, make sure that the following files have been downloaded at
+    ``/var/lib/clamav`` directory before starting the ClamAV daemon:
+
+    - bytecode.cvd
+    - daily.cld or daily.cvd
+    - main.cvd
+
+    These files can be downloaded from the artefactual-labs/clamav-files
+    repository, running:
```
Comment (sevein) [inline admin-manual/installation-setup/installation/install-ubuntu.rst]: Another nitpick, `artefactual-labs/clamav-files` could be a hyperlink too.
See http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets.

**Review by sevein — CHANGES_REQUESTED** [review]

File: admin-manual/installation-setup/installation/install-centos.rst [hunk 8]
```diff
@@ -270,9 +270,55 @@ Installation instructions
...
+
+     If the deemon has not started, please check the above section.
```
Comment (sallain) [inline admin-manual/installation-setup/installation/install-centos.rst]: Deemon should be daemon (though deemon sounds a lot more cute). It appears a few times below, as well.

File: admin-manual/installation-setup/installation/install-ubuntu.rst [hunk 9]
```diff
@@ -324,12 +365,53 @@ Ubuntu 14.04 (Trusty) installation instructions
...
+
+       sudo systemctl is-active clamav-daemon
+
+    When the service is up, the output is:
+
+    .. code:: bash
+
+       active
+
+    If the deemon has not started, please check the above section.
```
Comment (sallain) [inline admin-manual/installation-setup/installation/install-ubuntu.rst]: Which above section? Hyperlinking to it by name would be great.

**Review by sallain — CHANGES_REQUESTED** [review]

**Comment by sevein** [comment]
@mamedin I think that your branch has been deleted because you pushed it to the public remote.

**Comment by sevein** [comment]
I've just rescued it for you using the PR's remote that GH provides (if you're curious, you can do that following the instructions here: https://help.github.com/articles/checking-out-pull-requests-locally/).

**Comment by sevein** [comment]
@mamedin I wanted to try to push most recent signature files to the clamav-files repo but I've learned that the files are already too big to be managed by GH repos without lfs.

What I've done instead is to create a new release that I've called `20180428` and uploaded the files to it from the command line interface. See https://github.com/artefactual-labs/clamav-files/issues/4 for more details.

I have new snippets that you may want to try:

For CentOS:

```
yum install -y wget
wget -nv -nc -P /var/lib/clamav \
    https://github.com/artefactual-labs/clamav-files/releases/download/20180428/main.cvd \
    https://github.com/artefactual-labs/clamav-files/releases/download/20180428/daily.cvd \
    https://github.com/artefactual-labs/clamav-files/releases/download/20180428/bytecode.cvd
chown clamupdate:clamupdate /var/lib/clamav/*cvd
```

For Ubuntu:

```
wget -nv -nc -P /var/lib/clamav \
    https://github.com/artefactual-labs/clamav-files/releases/download/20180428/main.cvd \
    https://github.com/artefactual-labs/clamav-files/releases/download/20180428/daily.cvd \
    https://github.com/artefactual-labs/clamav-files/releases/download/20180428/bytecode.cvd
chown clamav:clamav /var/lib/clamav/*cvd
```

**Comment by mamedin** [comment]
@sallain , @sevein thanks very much for your suggestions. I have changed all that you said in your comments. This PR is ready again for CR

**Review by sevein — APPROVED** [review]

**Review by sallain — APPROVED** [review]

**Pull request merged**